src/Console/Command/ResultsCommand.php

```diff
@@ -0,0 +1,333 @@
+<?php
+
+namespace Lmc\Steward\Console\Command;
+
+use Lmc\Steward\Console\CommandEvents;
+use Lmc\Steward\Console\Event\BasicConsoleEvent;
+use Lmc\Steward\Process\ProcessWrapper;
+use Lmc\Steward\Publisher\AbstractPublisher;
+use Lmc\Steward\Publisher\XmlPublisher;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\TableCell;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Helper\TableStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Show results of the latest test run (according to the results.xml file)
+ */
+class ResultsCommand extends Command
+{
+    const OPTION_RESULTS_FILE = 'results-file';
+
+    /**
+     * Configure command
+     */
+    protected function configure()
+    {
+        $this->setName('results')
+            ->setDescription('Show test results overview')
+            ->addOption(
+                self::OPTION_RESULTS_FILE,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Path to test results xml file',
+                realpath(STEWARD_BASE_DIR . '/logs/' . XmlPublisher::FILE_NAME)
+            );
+
+        $this->getDispatcher()->dispatch(CommandEvents::CONFIGURE, new BasicConsoleEvent($this));
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int|null
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $filePath = $input->getOption(self::OPTION_RESULTS_FILE);
+
+        if (!is_readable($filePath)) {
+            throw new \RuntimeException(
+                sprintf(
+                    'Cannot read results file "%s", make sure it is accessible '
+                    . '(or use --results-file option if it is stored elsewhere)',
```

Comment:
BTW thanks unit tests, I swapped the arguments order while doing this :-)) 
https://travis-ci.org/lmc-eu/steward/jobs/127921608#L321